manifests/system.pp

```diff
@@ -74,15 +74,15 @@
   }
 
   # the fact won't work until rvm is installed before puppet starts
-  if getvar('::rvm_version') and !empty($::rvm_version) {
-    if ($version != undef) and ($version != present) and ($version != $::rvm_version) {
+  if $facts['rvm_version'] and !empty($facts['rvm_version']) {
```

Comment:
I think this may fail if the fact is not set. Perhaps it should be using `if fact('rvm_version')`?

Reply:
no, it returns undef and won't fail.